_locales/en/messages.json

```diff
@@ -315,6 +315,15 @@
     "message": "Enable",
     "description": "Label for the button to enable a style"
   },
+  "excludeStyleByDomainLabel": {
+    "message": "Exclude the current domain"
+  },
+  "excludeStyleByUrlLabel": {
+    "message": "Exclude the current URL"
+  },
+  "excludeStyleByUrlRedundant": {
+    "message": "The current URL is the domain page"
```

Comment:
I'm not sure if this sentence is correct.

Reply:
Looks good enough to me. =)